calls.go

```diff
@@ -82,14 +106,42 @@ func (call *callsCmd) list(ctx *cli.Context) error {
 		App:     app,
 		Context: context.Background(),
 	}
-	if ctx.Args().Get(1) != "" {
-		route := ctx.Args().Get(1)
-		params.Route = &route
+	if ctx.String("cursor") != "" {
+		cursor := ctx.String("cursor")
+		params.Cursor = &cursor
 	}
+	if ctx.String("path") != "" {
+		route := ctx.String("path")
+		params.Path = &route
+	}
+	if ctx.String("from-time") != "" {
+		fromTime := ctx.String("from-time")
+		fromTime_int64, err := time.Parse(time.RFC3339, fromTime)
+		if err != nil {
+			return err
+		}
+		res := fromTime_int64.Unix() / int64(time.Second)
```

Comment:
i think we can remove all the `/ int64(time.Second)`, the api for `Unix()` already returns exactly what we want https://golang.org/pkg/time/#Time.Unix and doing the division will end up changing the time to a different time: https://play.golang.org/p/t6dmjESpV2

I'm still wary of RFC3339 vs unix seconds in api (not just for reasons like this). generating unix time in shell is really easy: `date +%s` and also easy to use `bc` to offset it by 1 hour or whatever, and I'm not sure how flexible the RFC3339 date parser is in go.